packages/salesforcedx-vscode-core/src/orgPicker/orgList.ts

```diff
@@ -16,13 +16,6 @@ import { workspaceContext } from '../context';
 import { nls } from '../messages';
 import { OrgAuthInfo } from '../util';
 
-export interface FileInfo {
-  scratchAdminUsername?: string;
-  isDevHub?: boolean;
-  username: string;
-  devHubUsername?: string;
-  expirationDate?: string;
-}
```

Comment:
weird this we just hanging out unused.

Reply:
When this module used to read the config file directly, it used this type.  It could have been removed then but was missed.